README.md

```diff
@@ -10,9 +10,9 @@
 Build | Status
 -- | --
 Test coverage | [![codecov](https://codecov.io/gh/gazebosim/gz-fuel-tools/tree/main/graph/badge.svg)](https://codecov.io/gh/gazebosim/gz-fuel-tools/tree/main)

```

Comment:
This is still pointing to main

Reply:
Fixed in [b4b815a](https://github.com/gazebosim/gz-fuel-tools/pull/473/commits/b4b815ab7753cfedb97dbddad144801555b1cad5)